profiles/package.mask

```diff
@@ -32,6 +32,11 @@
 
 #--- END OF EXAMPLES ---
 
+# Wolfgang E. Sanyer <WolfgangESanyer@Gmail.com> (2021-03-11)
+# Per upstream, this has been deprecated in favour if citeproc. bug #775545
+# opened per the examples - Masked for removal in 30 days
+dev-haskell/pandas-citeproc
```

Comment:
I think you spelled it wrong.

Reply:
Fixed - I was thinking of the python `pandas` library for some reason, I have no idea why :-P